app/events/email-incoming/index.ts

```diff
@@ -85,8 +86,10 @@ module.exports.handler = createTriggerHandler(
       await sendEmail({
         fromName,
         recipient: userEmail,
-        subject: 'GCN Circular Submission Warning: Missing permissions',
-        body: 'You do not have the required permissions to submit GCN Circulars. If you believe this to be a mistake, please fill out the form at https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn, and we will look into resolving it as soon as possible.',
+        subject: submissionFailedSubject('Not an Authorized Submitter'),
+        body: submissionFailedBody(
+          'The email address you are submitting this circular from is either not an approved legacy submitter or you have not completed the peer endorsement process, which authorizes you to submit GCN Circulars. If you believe this to be a mistake, please fill out the form at https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn, and we will look into resolving it as soon as possible.'
```

Comment:
"The email address you are submitting this circular from is either not an approved legacy submitter or you have not completed the peer endorsement process, which authorizes you to submit GCN Circulars."

This is a very long sentence. Let's break it up a bit. Also the error message should tell the user what steps they need to take to fix it.